reducers/gotPlayer/player.js

```diff
@@ -39,8 +39,8 @@ export const player = {
   getProfile: state => player.getPlayer(state).profile,
   getPlayerName: state => player.getPlayer(state).profile.personaname,
   getLastLogin: state => player.getPlayer(state).profile.last_login,
-  getWins: state => player.getPlayer(state).win,
-  getLosses: state => player.getPlayer(state).lose,
+  getWins: state => state.yaspReducer.gotPlayer.winloss.win,
+  getLosses: state => state.yaspReducer.gotPlayer.winloss.lose,
   getMmrEstimate: state => player.getPlayer(state).mmr_estimate,
```

Comment:
Actually, let's just leave it like this for now. It's sort of weird to import wins/losses here, this should be responsible for all info associated player I think.